src/sentry/static/sentry/app/views/groupActivity/noteInput.jsx

```diff
@@ -70,6 +70,11 @@ var NoteInput = React.createClass({
     this.setState({value: e.target.value});
   },
 
+  onKeyDown(e) {
+    // Auto submit the form on [meta] + Enter
+    e.keyCode === 13 && e.metaKey && this.onSubmit(e)
```

Comment:
Also, fwiw, I originally did `e.target.form.submit()` like normal JavaScript people would do, and the page reloaded and the form never submit. So I assume that doesn't work for some reason with React.